### PR TITLE
Changed: remove "self" via SwiftFormat script

### PR DESCRIPTION
**Pull request by minarikg**
changed: scripts/format.sh
the rest is done by SwiftFormat

### CODE REVIEW

File: Source/Core/EventEmitter.swift [hunk 1]
```diff
@@ -19,13 +19,13 @@ class ReceiverHandle<Parameters>: Hashable {
 
     fileprivate init(object: AnyObject, handler: @escaping (AnyObject) -> (Parameters) -> Void) {
         self.object = object
```
Comment (aliak00) [inline Source/Core/EventEmitter.swift]: 😬

**Comment by minarikg** [comment]
The reason for this to highlight the cases when self is retained by a closure. And think about potential retain cycles when you see “self”.
Xcode also does color highlights for properties vs local variables. Also functions should be simple and short enough to realize which are local variables.

File: Source/Core/EventEmitter.swift (hunk 1 above)
Comment (minarikg) [reply Source/Core/EventEmitter.swift]: This is normal for an init in Swift.
or like ‘guard let self = self else {return}’ in closures.

**Comment by aliak00** [comment]
> The reason for this PR is to highlight the cases when self is retained by a closure. And think about potential retain cycles when you see “self”.

Ah! That's a fair point. I'm not sure that's something people regularly think about when they see `self` but I see the advantage in that indeed.

>Xcode also does color highlights for properties vs local variables. Also functions should be simple and short enough to realize which are local variables.

Code changes hardly apply to an entire function. And while i agree with the theory whole heartedly, "simple and short" is sometimes just not practical sometimes :)

**Pull request merged**